mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/work/SearchWorkspace.java

```diff
@@ -98,9 +99,10 @@ public interface SearchWorkspace {
 	 * <p>
 	 * Note that some operations may still be waiting in a queue when {@link #refresh()} is called,
 	 * in particular operations queued as part of automatic indexing before a transaction is committed.
-	 * These operations will not be applied immediately just because  a call to {@link #refresh()} is issued:
+	 * These operations will not be applied immediately just because a call to {@link #refresh()} is issued:
 	 * the "refresh" here is a very low-level operation handled by the backend.
 	 */
+	// TODO: Yoann, does it make sense to talk about "automatic indexing and transaction being committed" in a context of a pojo mapper?
```

Comment:
It does not, feel free to remove the paragraph starting with "Note that some operations [...]".